default/index.js

```diff
@@ -59,18 +64,32 @@ function readDirectory(dirName, stream) {
   return files;
 }
 
-// If stream is true, a stream is provided instead of a buffer for
-// each file object along with the corresponding size
-function getDefaultProjects(stream) {
-  var DefaultProjects = {};
+function cacheProjectFiles() {
   var projects = fs.readdirSync(__dirname);
   projects.splice(projects.indexOf("index.js"), 1);
```

Comment:
Gideon used this to remove the file we're looking at from the directory listing, leaving just the folders containing default projects in the array.